packages/react-router/src/components/Link/tests/Link.test.tsx

```diff
@@ -43,10 +43,9 @@ describe('<Link />', () => {
 
     it('sets the `to` prop and passes along other props to ReactRouterLink', () => {
       const url = '/home';
-      const props = {className: 'foo'};
```

Comment:
This refactoring surfaces the problem where it complains "className is not a valid prop" if you don't make the change to Link.tsx